bootstrapgenerator/src/main/java/nl/openweb/hippo/groovy/ScriptClassFactory.java

```diff
@@ -112,10 +113,10 @@ public static ScriptClass getInterpretingClass(final File file, final boolean ke
     public static String readFileEnsuringLinuxLineEnding(final File file) throws IOException {
         String content = FileUtils.fileRead(file);
         if (content.contains(LINE_END_MAC)) {
-            return content.replaceAll(LINE_END_WINDOWS, LINE_END_LINUX)
+            content = content.replaceAll(LINE_END_WINDOWS, LINE_END_LINUX)
                     .replaceAll(LINE_END_MAC, LINE_END_LINUX);
         }
-        return content;
+        return content.replaceAll("[\\t ]+" + NEWLINE, NEWLINE);
```

Comment:
replacing tabs for NewLine? Won't this increase the amount of new lines compared to the original